app/services/user_service.rb

```diff
@@ -74,9 +83,7 @@ def login(email:, password:)
 
     # Creates a new provider credential, and adds it to the User's provider
     # credentials array
-    def create_provider_credential!(
-      user_id: nil, id: nil, email: nil, api_token: nil, full_name: nil
-    )
+    def create_provider_credential!(user_id: nil, id: nil, email: nil, api_token: nil, full_name: nil)
```

Comment:
Should these go on new lines?

Reply:
(honest q)

Reply:
Ahh, it wasn't over 120 characters on the line, so I left it as is. I could break it up if you'd like 👍